File: WantToSell.Application/Models/Paging/Pager.cs
namespace WantToSell.Application.Models.Paging;

public class Pager
{
    public int PageIndex { get; set; } = 1;
    public int PageSize { get; set; } = 20;
    public string SortColumn { get; set; } = "DateCreatedUtc";
    public bool Ascending { get; set; } = true;
}

File: WantToSell.WebClient/want-to-sell-client/src/app/common/Pager.ts
export class Pager
{
  public pageIndex: number;
  public pageSize: number;
  public sortColumn: string;
  public ascending: boolean;

  constructor(pageIndex: number, pageSize: number, sortColumn: string, ascending: boolean) {
    this.pageIndex = pageIndex;
    this.pageSize = pageSize;
    this.sortColumn = sortColumn;
    this.ascending = ascending;
  }
}
